go/ct/evm_fuzz_test.go

```diff
@@ -0,0 +1,285 @@
+// Copyright (c) 2024 Fantom Foundation
+//
+// Use of this software is governed by the Business Source License included
+// in the LICENSE file and at fantom.foundation/bsl11.
+//
+// Change Date: 2028-4-16
+//
+// On the date above, in accordance with the Business Source License, use of
+// this software will be governed by the GNU Lesser General Public License v3.
+
+package ct_test
+
+import (
+	"fmt"
+	"math"
+	"testing"
+
+	"pgregory.net/rand"
+
+	"github.com/Fantom-foundation/Tosca/go/ct"
+	. "github.com/Fantom-foundation/Tosca/go/ct/common"
+	"github.com/Fantom-foundation/Tosca/go/ct/st"
+	"github.com/Fantom-foundation/Tosca/go/vm"
+	"github.com/Fantom-foundation/Tosca/go/vm/geth"
+	"github.com/Fantom-foundation/Tosca/go/vm/lfvm"
+)
+
+// FuzzGeth is a fuzzing test for the geth EVM implementation
+// TODO: it would be interesting to have a method to extend the corpus of the fuzzer
```

Comment:
An idea for this (both for follow-up work, not this PR): run the fuzzer on a list of EVMs instead of a single one; this way, you only have one entry point and the corpus is shared

Reply:
Yes, I thought about it. I would have to run some experiment to see how does this scale, and it is nice to use when a bug is found.
Another competing idea is to link or copy found errors corpus from `/testdata/FuzzingTest1/datahash` to testdata/fuzzingTest2/datahash`, since they follow the same layout they should be easily exchangeable between any crash test, and diff tests. Such files are generated when an error is found, I guess they should be committed, I have a couple of unsolved ones which I will commit once evmzero bugs are fixed.